eisbuk-admin/src/store/actions/customerOperations.ts

```diff
@@ -79,28 +83,60 @@ export const updateCustomer = (
  * @param customer to delete from firestore
  * @returns async thunk
  */
-export const deleteCustomer = (
-  customer: CustomerBase
-): FirestoreThunk => async (dispatch) => {
-  try {
-    const db = getFirestore();
-    const docRef = doc(db, getCustomersCollPath(), customer.id);
-
-    await setDoc(docRef, { deleted: true }, { merge: true });
-
-    dispatch(
-      enqueueNotification({
-        key: new Date().getTime() + Math.random(),
-        message: `${customer.name} ${customer.surname} ${i18n.t(
-          NotificationMessage.Removed
-        )}`,
-        closeButton: true,
-        options: {
-          variant: NotifVariant.Success,
-        },
-      })
-    );
-  } catch {
-    dispatch(showErrSnackbar);
-  }
-};
+export const deleteCustomer =
+  (customer: CustomerBase): FirestoreThunk =>
+  async (dispatch) => {
+    try {
+      const db = getFirestore();
+      const docRef = doc(db, getCustomersCollPath(), customer.id);
+
+      await setDoc(docRef, { deleted: true }, { merge: true });
+
+      dispatch(
+        enqueueNotification({
+          key: new Date().getTime() + Math.random(),
+          message: `${customer.name} ${customer.surname} ${i18n.t(
+            NotificationMessage.Removed
+          )}`,
+          closeButton: true,
+          options: {
+            variant: NotifVariant.Success,
+          },
+        })
+      );
+    } catch {
+      dispatch(showErrSnackbar);
+    }
+  };
+
+interface BookingMailProps {
+  to: string;
+  subject: string;
+  accessLink: string;
+}
+
+export const sendBookingsLink =
+  ({ to, accessLink, subject }: BookingMailProps): FirestoreThunk =>
+  async (dispatch) => {
+    try {
+      const newEmail: EmailMessage = {
+        to,
+        message: { subject, html: accessLink },
```

Comment:
We'll need to add some template here to put some boilerplate in the email. It can go in as is for now, but this is something we'll need soon.

The template needs access to the athlete object to be able to write `Dear John Doe, here is your link`. So either we pass the full message to this function (in this case the `accessLink` should be renamed to `emailBody`) or we pass the athlete object too so that this function has all it needs to compose the email.

Reply:
I'm more down with the second solution. Instead of passing `accessLink`, we will pass customer id and the rest can be generated from store entry for customer keyed by customer id.

On it!